src/components/Navbar/index.tsx

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+
+import Search from "./_search";
```

Comment:
Why the component name is starting with underscore?

Reply:
I got the suggestion that it is better to use. to indicate that I will use it in the with other component. but actually I didn't understand the basic difference. I will change it, thanks